index.js

```diff
@@ -105,12 +90,16 @@ app.use(
 app.use(
   session({
     secret: process.env.SESSION_KEY || "meowmeow",
+    resave: true,
+    saveUninitialized: true
   })
 );
 app.use(flash());
 app.use(busboy());
-app.use(bodyparser.urlencoded());
-
+app.use(express.json());
+app.use(express.urlencoded({
+  extended: true
```

Comment:
I feel like `extended` is not necessary as it isn't used yet. We can enable it when it is needed.